CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * EXODistributionGroup
   * Fixes the export of group membership to use Identity.
+* TeamsGroupPolicyAssignment
+  * Fixes the export of CsGroup, when the display name of a group is included in another display name. Fix [#3736](https://github.com/microsoft/Microsoft365DSC/issues/3736)
```

Comment:
A very minor thing. Could you please put "Fix [..." in a new line?

Reply:
Yes